src/shared/components/enhancedReactTable/IEnhancedReactTableProps.ts

```diff
@@ -12,11 +12,16 @@ export interface IEnhancedReactTableProps {
     headerControlsProps?: ITableExportButtonsProps;
     columns?: IColumnDefMap; // column definitions (including component renderers)
     rawData: Array<any>; // raw data
+    initItemsPerPage?: number; // initial number of items per page
+    initPage?:number; // initial page
+    itemsName?:string; // name of items, e.g. "mutations"
```

Comment:
`itemsName` doesn't need to be a property of `EnhancedReactTable`, it might be better to pass this in the `headerControlsProps`.